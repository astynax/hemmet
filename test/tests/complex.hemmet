form:search-form$theme>input.query^red-text>(div.help~hidden_t)+span.submit&button~text_small>.hint
