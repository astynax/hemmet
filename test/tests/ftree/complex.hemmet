file1 folderA/{fileA1 folderAA/ folderAB/{fileAB1 fileAB2} fileA2} folderB/folderBA/folderBAA/fileBAA1 file2
