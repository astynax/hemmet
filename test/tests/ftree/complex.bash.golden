#!/bin/bash
set -euf -o pipefail

touch "file1"
touch "file2"
mkdir "folderA" && pushd "folderA"
  touch "fileA1"
  touch "fileA2"
  mkdir "folderAA"
  mkdir "folderAB" && pushd "folderAB"
    touch "fileAB1"
    touch "fileAB2"
  popd
popd
mkdir "folderB" && pushd "folderB"
  mkdir "folderBA" && pushd "folderBA"
    mkdir "folderBAA" && pushd "folderBAA"
      touch "fileBAA1"
    popd
  popd
popd
